.mailmap

```diff
@@ -3,6 +3,8 @@ Aaron Heckmann <aaron.heckmann@gmail.com> <aaron.heckmann+github@gmail.com>
 Aayush Ahuja <aayush.a@directi.com> aayush.a <aayush.a@directi.com>
 Abe Fettig <abefettig@gmail.com> <abe@fettig.net>
 Akito Ito <akito0107@gmail.com> <akito_ito@r.recruit.co.jp>
+Alejandro Estrada <estrada9166@gmail.com>
+Alejandro Estrada <estrada9166@gmail.com> <estrada9166@hotmail.com>
```

Comment:
Duplicate

Reply:
It has to be like that AFAICT. First entry for the name (it is lowercase in the commit log). Second entry for the email alias.